requirements.txt

```diff
@@ -1,3 +1,4 @@
 anymarkup
 lockfile
 jsonpointer
+requests
```

Comment:
Good that it's added to `requirements.txt`, however, we will need to add this to each `Dockerfile` in regards to pulling in the respective `python-requests` yum/apt-get dependencies. Have a look at how we import anymarkup, lockfile, jsonpointer, etc to our `Dockerfile` :)